doc/modbus_rtu_get_suppress_echo.txt

```diff
@@ -0,0 +1,48 @@
+modbus_rtu_get_suppress_echo(3)
+===============================
+
+
+NAME
+----
+modbus_rtu_get_suppress_echo - get local echo suppression status
+
+
+SYNOPSIS
+--------
+*int modbus_rtu_get_suppress_echo(modbus_t *'ctx');*
+
+
+DESCRIPTION
+-----------
+
+The _modbus_rtu_get_suppress_echo()_ function shall return the current
+enabled/disable state of local echo suppression workaround of
+the libmodbus context 'ctx'.
+
+This function can only be used with a context using a RTU backend.
+
+
+RETURN VALUE
+------------
+The _modbus_rtu_get_suppress_echo()_ function shall return 0 if workaround
+is disable, 1 when it is enabled.
+Otherwise it shall return -1 and set errno.
```

Comment:
is it really necessary to do this? can't it just be a bool and have it be false for non-rtu backends?

Reply:
See above, would work for me, too.